CAM Platform/organizations-api.md

```diff
@@ -1,42 +1,60 @@
 {{{
 "title": "Organizations API",
-"date": "09-01-2016",
-"author": "",
+"date": "10-30-2018",
+"author": "Arya Roudi",
 "attachments": [],
-"contentIsHTML": false
+"contentIsHTML": false,
+"keywords": ["api", "organization api", "manage api", "manage organization"]
 }}}
 
 ### Manage an Organization
 
 | Resource | Description |
 |----------|-------------|
-| GET /services/organizations/{organization_name} | Gets the schema of the given organization. |
-| PUT /services/organizations/{organization_name} | Updates an existing organization. |
-| PUT /organizations/{organization_name}/sync_groups | Queues a request to sync LDAP groups. |
+| [GET /services/organizations/{organization_name}](#get-servicesorganizationsorganization_name) | Get the schema of the given organization. |
```

Comment:
Hey @aryaroudi, I realized this one has not been updated to include links in the summary table. Could you please update? Thanks!